### DISCUSSION TITLE
插座功耗和耗电量显示单位不对，日耗电量月耗电量实体缺失。

### DISCUSSION THREAD

**Discussion by PilotEmerson** [XiaoMi/ha_xiaomi_home#442] [Ideas]
插座功耗和耗电量显示单位不对，日耗电量月耗电量实体缺失。

**Comment by JimmyKmi** [comment]
同
https://github.com/XiaoMi/ha_xiaomi_home/issues/325
https://github.com/XiaoMi/ha_xiaomi_home/issues/164
https://github.com/XiaoMi/ha_xiaomi_home/issues/89
https://github.com/XiaoMi/ha_xiaomi_home/issues/30
https://github.com/XiaoMi/ha_xiaomi_home/issues/424

提问前请先查询是否有相同问题

**Reply by Superxinxing** [reply]
电压显示也不对，小数点后三位了
